CMakeLists.txt

```diff
@@ -209,10 +209,7 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 
-if (NOT $ENV{S2N_LIBCRYPTO} MATCHES "awslc")
-    # add cast-qual back in for non AWS-LC
-    target_compile_options(${PROJECT_NAME} PRIVATE -Wcast-qual)
-endif()
+target_compile_options(${PROJECT_NAME} PRIVATE -Wcast-qual)
```

Comment:
Is there still a reason to have this flag separated from all of the other flags? I wonder if it should just be added to the rest of the options:
https://github.com/aws/s2n-tls/blob/ada65a76ac7154644f85024ed5ae4696f4409226/CMakeLists.txt#L137-L140

Reply:
The readability will be improved if we put the flag into all of this. I will add `-Wcast-qual` to this part.